grakn-test/src/test/java/ai/grakn/test/graql/reasoner/BenchmarkTests.java

```diff
@@ -140,7 +140,7 @@ public void testTransitiveChain()  {
         //results @N = 450 101475   ?     ?       ?     14s
         //results @N = 500 125250         ?       ?     21-26s
 
-        //with materialisation            IC    IC titan
+        //with materialisation            IC    IC janus
```

Comment:
Given these are old benchmark results, maybe we should keep it saying `titan`? I'm not totally sure.